PlateSpace/ETL/README.md

```diff
@@ -0,0 +1,38 @@
+# Yelp Import Tool
+
+The purpose of this script is to load restaurant data into a MongoDB Atlas database using the Yelp Fusion API.
+
+Getting Started:
+
+  1. Clone/download repository
+  1. Create a MongoDB Atlas cluster
+  1. Register yelp as a developer for getting an auth token.
+  1. Edit 'config.js' file:
+    - `MONGODB_ATLAS_URI`: here you need to put your mongoDB connection string.
+    - `YELP_AUTH_TOKEN`: here you need to put your OAuth2 bearer token generated from [https://api.yelp.com/oauth2/token](https://api.yelp.com/oauth2/token). More information is available in the [Yelp Developer site](https://www.yelp.com/developers/documentation/v3/authentication).
+  1. Install yarn using this link: https://yarnpkg.com/lang/en/docs/install/
+  1. Install the following packages using yarn: `yarn`
+  1. Run command `node index.js`
+
+The script creates new collection for restaurants, using Yelp Fusion API.
+
+Collection schema:
+    ```{
```

Comment:
I think you want a line break after the third backtick, so that the rendered markdown ends up on multiple lines instead of one line. Currently in the preview it looks like it's showing up on a single line.